internal/command/project_test.go

```diff
@@ -14,10 +15,12 @@ import (
 	"github.com/zitadel/zitadel/internal/id"
 	id_mock "github.com/zitadel/zitadel/internal/id/mock"
 	"github.com/zitadel/zitadel/internal/repository/project"
+	"github.com/zitadel/zitadel/internal/repository/user"
 	"github.com/zitadel/zitadel/internal/zerrors"
 )
 
 func TestCommandSide_AddProject(t *testing.T) {
+	t.Parallel()
```

Comment:
Have a look at the current PR's, if I remember correctly there is again one to remove all Parallel() calls?